src/neptune/internal/utils/dependency_tracking.py

```diff
@@ -36,11 +39,16 @@ def log_dependencies(self, run: "Run") -> None:
 class InferDependenciesStrategy(DependencyTrackingStrategy):
     def log_dependencies(self, run: "Run") -> None:
         dependencies = []
-        dists = list(sorted(distributions(), key=lambda d: d.metadata["Name"]))
+
+        def sorting_key_func(d: Distribution) -> str:
+            _name = d.metadata["Name"]
+            return _name.lower() if isinstance(_name, str) else ""
+
+        dists = sorted(distributions(), key=sorting_key_func)
 
         for dist in dists:
-            name, version = dist.metadata["Name"], dist.metadata["Version"]
-            dependencies.append(f"{name}=={version}")
+            if dist.metadata["Name"]:
```

Comment:
@AleksanderWWW - added this check to prevent `None==None` output in requirements.txt